forge-ai/src/main/java/forge/ai/SpecialCardAi.java

```diff
@@ -651,6 +651,18 @@ public static boolean consider(final Player ai, final SpellAbility sa) {
         }
     }
 
+    // Grothama, All-Devouring
+    public static class GrothamaAllDevouring {
```

Comment:
Since it has pretty good stats I'm not sure AI should be just as eager to get rid of it for some draw if it controls it

Reply:
Ah, true, I'll tweak that. You're referring to the case when the AI controls it and attacks with its own creatures, right?

Reply:
In that case , I'm not even sure what a universally good condition would be for that, I think I'll just make the AI not agree to fight its own Grothama with its own creatures at all